telemetry/specs/telemetry_report.mdk

```diff
@@ -511,11 +511,11 @@ particular deployment.
 
 * F (1b): Tracked Flow Association
   : Indicates that this telemetry report is for a tracked flow, i.e. the
-    packet matched a flow watchlist somewhere (in case of INT or IOAM) or
-    locally (in case of postcard). The report might include INT or IOAM
+    packet matched a flow watchlist somewhere (in case of INT-MD, INT-MX or IOAM) or
+    locally (in case of INT-MX). The report might include INT-MD or IOAM
```

Comment:
s/INT-MX/INT-XD
I had it wrong in my previous comment

Reply:
Done